doc/fileexplorer.rst

```diff
@@ -1,23 +1,84 @@
 #####
-Files
+Files 
 #####
 
-The **Files** pane is a built-in filesystem and directory browser allowing you to view and filter files and their properties, open them with the :doc:`editor` or an external tool, and perform common management tasks (varies by platform).
+The **Files** pane is a filesystem and directory browser built right into Spyder.
+You can view and filter files according to their type and extension, open them with the :doc:`editor` or an external tool, and perform many common operations.
 
 .. image:: images/files/files-standard.png
-   :alt: Spyder File Explorer panel, showing a tree view of files and metadata
+   :alt: Spyder Files pane, showing a tree view of files and metadata
 
 
 
-==========================
-Performing file operations
-==========================
+===============
+File operations
+===============
 
-You can use context menus to access a number of functions on a per-file and per-directory basis, including running scripts, creating, renaming, moving, editing or deleting files, performing basic operations with ``git``, and opening an external file explorer.
+To browse the files on your system, you can use the arrows at the top of the pane.
+You can expand/collapse the folders in the pane to display the files and subdirectories hierarchically. 
+Double-clicking a folder will open it, showing the files inside and making it your working directory.
 
-.. image:: images/files/files-contextmenu-new.png
-   :align: center
-   :alt: File Explorer context menu, with common file manipulation tasks
+.. image:: images/files/files-browse.gif
+   :alt: Spyder Files pane showing browsing directories
+
+
+To open a file in the :guilabel:`Editor` from the :guilabel:`Files` pane, double-click its name.
+If you right-click over it, you will see a context menu that allows you to access a number of functions, including running scripts; creating, renaming, moving, deleting files; and opening them in your computer's file manager.
+
+.. image:: images/files/files-context-menu.png
+   :alt: Spyder Files pane showing context menu
+
+You can copy and paste one or several files to and from the pane, and copy their absolute or relative paths to the clipboard as text.
+If copying the paths for multiple files, they will be automatically formatted so you can paste them directly into a Python list.
+
+.. image:: images/files/files-copy-path.gif
+   :alt: Spyder Files pane showing copying absolute path to Editor
+
+Additionally, the :guilabel:`Files` pane allows you to perform basic operations with ``git``, like commiting your changes and browsing the repository a given file or folder is part of.
+
+.. image:: images/files/files-commit.gif
```

Comment:
There appears to be about a second of black slug at the end of the GIF. Instead, it should use that time to pause on the completed commit to show the result of the action. Otherwise, looks good.

Reply:
Agree with @CAM-Gerlach too.

Reply:
Agreed

Reply:
Still doesn't pause at the end for more than a few hundred milliseconds, should pause for a second so the user can see the final result